catalog-rest-service/src/main/resources/ui/src/pages/login/index.tsx

```diff
@@ -44,13 +44,19 @@ const SigninPage = () => {
       }
       case AuthTypes.OKTA: {
         return (
-          <button className="tw-signin-button tw-text-white tw-bg-blue-700 hover:tw-bg-blue-600">
-            Sign in with Okta
+          <button className="tw-signin-button">
```

Comment:
We can use local variable like icon and text and at the end just returned button component instead of return for each switch case.

Reply:
Done